src/module/overrideModule.js

```diff
@@ -39,21 +39,32 @@ var OverrideModule = class OverrideModule {
         const _shouldAnimateActor = WindowManager.prototype._shouldAnimateActor;
         WindowManager.prototype._shouldAnimateActor = function () {
             let actor = arguments[0];
+            let types = arguments[1];
+
+            if (
+                !actor ||
+                this._skippedActors.delete(actor) ||
+                !this._shouldAnimate() ||
```

Comment:
Ah, true. I remember I saw that part but I didn't pay attention to the returned value. Interesting, I didn't notice anything weird in my tests because of this.